scss/dijit/layout/_tabContainer.scss

```diff
@@ -28,6 +28,10 @@
         padding: $tab-padding;
         overflow: visible;
       }
+
+      .rules__tab-container .dijitContentPane{
```

Comment:
This change can't be here, this file is just for the dijit theme and this change it's related just to the "rules" tab in a specific page, so:

1. Create a ```_rules.css``` here: https://github.com/dotCMS/dotcms-dojo-theme/tree/4bc5786a67806a8fac7789e4f6e58ee4c9ac8f27/scss/backend/dot-admin/components

2. Add the change there

3. Don't forget to import the new file here: https://github.com/dotCMS/dotcms-dojo-theme/blob/4bc5786a67806a8fac7789e4f6e58ee4c9ac8f27/scss/backend/dot-admin/components/_common.scss